tldextract/tests/main_test.py

```diff
@@ -36,29 +36,23 @@ def assert_extract(
         assert expected_domain == ext.domain
         assert expected_tld == ext.tld
 
-
```

Comment:
you mean the number of lines in between functions?  yeah, i thought it was one, but that is in between methods.  i'll make them twos.